modules/capability/go.mod

```diff
@@ -0,0 +1,169 @@
+module github.com/cosmos/ibc-go/modules/capability
+
+go 1.19
+
+replace github.com/cosmos/ibc-go/v7 => ../../
```

Comment:
Why does capability module reference ibc-go? I guess for tests? Will this cause versioning issues?